share/spice/thesaurus/thesaurus.js

```diff
@@ -6,16 +6,29 @@
             return Spice.failed("thesaurus");
         }
 
+        // Check if user want to find antonym only
+        var searchAntonymOnly = (document.title.indexOf('antonym') > -1);
```

Comment:
@nkhanhtrn we have the original query below in the `query` variable -- it's much better to check that instead of the page title